src/MenuActiveEffects.cpp

```diff
@@ -167,6 +223,14 @@ TooltipData MenuActiveEffects::checkTooltip(const Point& mouse) {
 				tip.addText(ss.str());
 			}
 
+			if(effect_icons[i].type != EFFECT_SHIELD){
+				std::stringstream ss2;
+				if(effect_icons[i].stacks > 1){
+					ss2 << "x" << effect_icons[i].stacks << " stacks";
```

Comment:
To make this string able to be translated use this instead:

```
msg->get("%dx stacks", effect_icons[i].stacks);
```

You should also do something similar for the string that's rendered on top of the icon. Different languages might represent that differently.